benchmark/streamly-benchmarks.cabal

```diff
@@ -426,6 +435,7 @@ benchmark Memory.Array
 benchmark FileSystem.Handle
   import: bench-options
   type: exitcode-stdio-1.0
+  ghc-options: +RTS -M3000M -RTS
```

Comment:
Weren't we specifying memory options via the scripts?

Reply:
This for restricting memory consumed by GHC when compiling benchmarks. The one you are talking about heap/memory when running benchmarks.